### ISSUE TITLE
Redox does not render, failed to open image /etc/redoxfs.bin: I/O error

### ISSUE THREAD

**Issue by Nogbit** [redox-os/redox#568]
**Reproduction**: Followed install steps from the readme

**Expected behavior**: Expected Redox to fully render

**Actual behavior**: Redox does start, and I'm able to log in, but, no GUI is ever presented.

**Build information**: 

```Shell
[marc@liberty redox]$ rustc -V
rustc 1.9.0-nightly (02310fd31 2016-03-19)

[marc@liberty redox]$ git rev-parse HEAD
e9982dc68680e76dc472c2135c6152e8775b457f

[marc@liberty redox]$ qemu-i386 -version
qemu-i386 version 2.5.0, Copyright (c) 2003-2008 Fabrice Bellard

[marc@liberty redox]$ uname -a
Linux liberty 4.1.19-1-MANJARO #1 SMP PREEMPT Sat Mar 5 23:03:00 UTC 2016 x86_64 GNU/Linux
```

**Misc**: Here is the output of running ``make qemu``.

------

```Shell
[marc@liberty redox]$ make qemu
#TODO: filesystem/bin/env filesystem/bin/test
CARGO_TARGET_DIR=build RUSTC="./rustc-i386.sh" cargo rustc --manifest-path crates/ion/Cargo.toml --bin ion-shell --verbose --target=i386-unknown-redox.json -- -L build/i386-unknown-redox/debug -C no-prepopulate-passes -C no-stack-check -C opt-level=2 -Z no-landing-pads -A dead_code
       Fresh peg v0.3.9 (https://github.com/crazymykl/rust-peg.git?branch=libsyntax-changes-again#1efaca63)
       Fresh glob v0.2.11
       Fresh ion-shell v0.1.2 (file:///home/marc/Documents/projects/redox/crates/ion)
find filesystem -type f -o -type l | cut -d '/' -f2- | sort | awk '{printf("file %d,\"%s\"\n", NR, $0)}' > build/i386-unknown-redox/debug/filesystem.gen
nasm -f bin -o build/i386-unknown-redox/debug/harddrive.bin -l build/i386-unknown-redox/debug/harddrive.list -D ARCH_i386 -D TIME="`date "+%F %T"`" -ibuild/i386-unknown-redox/debug/ -ikernel/ -ifilesystem/ kernel/harddrive.asm
qemu-system-i386 -serial mon:stdio -m 1024 -d guest_errors -enable-kvm -vga std -usb -device usb-tablet -drive file=build/i386-unknown-redox/debug/harddrive.bin,format=raw,index=0,media=disk -net nic,model=rtl8139 -net user -net dump,file=build/i386-unknown-redox/debug/network.pcap
Redox 32 bits
MADT: Unknown entry type: 4, length 6
HPET: Unknown Table
Extra: 0
Extra: FA
Extra: FA
Primary Master: Status: 58 Serial: QM00001 Firmware: 2.5+ Model: QEMUHARDDISK Size: 62 MB
Primary Slave: Status: 0
Secondary Master: Status: 41 Error: 2
Secondary Slave: Status: 0
IDE Primary Master: Redox Filesystem
UHCI on: C100, IRQ: B
 CMD 00 to 00 to 00 STS 20 INTR 00 FRNUM 64 to 00 FLBASEADD 00 to D16000 CMD 00 to 01
 PORTSC1 83 to 0283 to 83
 Device Found 83 to 87
Manufacturer: QEMU
Product: QEMU USB Tablet
Serial: 42
 PORTSC2 80 to 0280 to 80
RTL8139 on: C000 port mapped IRQ: 0B MAC: 52.54.0.12.34.56 IMR: 05 CMD: 0D RCR: 9E TCR: 77800000
Starting HID driver
Disk I/O Error: No such file or directory (os error 2)
redoxfs: failed to open image /etc/redoxfs.bin: I/O error
`orbital` to start GUI

redox login: marc
ion:file:/home/# ls
.ion_history
.ionrc
LICENSE.md
README.md
colors.lua
example.sh
rust.bmp
schemabanner.bmp
test.lua
test.txt
test.wav
ion:file:/home/# 
```

**Comment by DomThePorcupine** [comment]
Did you try running `orbital`?

**Comment by genodeftest** [comment]
I think @Nogbit mixed up 2 different things:
1. the error message about redoxfs.bin not found
2. that you need to type `orbital` + press the enter key to start a GUI.

**Comment by Nogbit** [comment]
I will give that a shot tonight, the readme doesn't mention that.

**Comment by genodeftest** [comment]
> failed to open image /etc/redoxfs.bin: I/O error
Is now fixed by #570

**Comment by Nogbit** [comment]
Thanks all, running ``orbital`` did the trick!
